activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADJwtFilterProperties.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.autoconfigure.aad;
+
+import org.hibernate.validator.constraints.NotEmpty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import java.util.List;
+
+@Validated
+@ConfigurationProperties("azure.activedirectory")
+public class AzureADJwtFilterProperties {
```

Comment:
Please reference this commit https://github.com/Microsoft/azure-spring-boot-starters/commit/86aa124ebc27de7c99501e3153e7d664c51fd824

to add Java doc for each field. There will be hints when user use the properties.